src/GitTools.IssueTrackers/Context/IssueTrackerContext.cs

```diff
@@ -1,29 +0,0 @@
-﻿namespace GitTools.IssueTrackers
-{
-    public class IssueTrackerContext : Disposable, IIssueTrackerContext
-    {
-        public IssueTrackerContext()
```

Comment:
GitVersion manages to have a single context which contains contextual information like current branch and such. Arguments are irrelevant after bootstrapping.

And this sort of context can be passed around, it doesn't need to own other things?

We *may* need disposable contexts in one of the libraries/consumers of this library, but in GitTools.IssueTrackers we have no disposable resources so no point making things which do not need to be disposed, IDisposable